### ISSUE TITLE
Issue with imported dxf file 

### ISSUE THREAD

**Issue by Franckapik** [vilemduha/blendercam#170]
Hi! I would like to understand why i have an error on cutout blendercam. 
The shape of my imported dxf seem's to be wrong ... :/
https://pasteboard.co/JFp2hlF.png
and this is the error : https://pasteboard.co/JFp2Mjr.png
I have clean up with remove double and merge by distance
but it seem's to have another issue...
Is there a command to be sure that every vertex are merged ? Someone had an idea of what is causing this issue ?

Thank you !

**Comment by Franckapik** [comment]
Here is my blend file : https://pasteall.org/blend/9f65fbeda3ca479a80f8ea2655217e90

**Comment by TurBoss** [comment]
hello,
there is a unmerged vertex at the bottom left
i and only after creating faces for each part it allowed me to calculate

![Screenshot from 2020-12-18 00-08-16](https://user-images.githubusercontent.com/184088/102554456-319c6480-40c5-11eb-94fc-27566ed4db86.png)
![Screenshot from 2020-12-18 00-08-05](https://user-images.githubusercontent.com/184088/102554458-33662800-40c5-11eb-8faa-67ce4ed02250.png)


[pirate.zip](https://github.com/vilemduha/blendercam/files/5712616/pirate.zip)
